<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>onlineGame</class>
 <widget class="QDialog" name="onlineGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202030;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cRoomBtn">
         <property name="minimumSize">
          <size>
           <width>56</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Imprint MT Shadow</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    color: #b1b1b1;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 3px;
    border-color: #0023F5;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 38px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

QComboBox
{
    selection-background-color: #ffaa00;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);

    border: 1px solid #1e1e1e;
    border-radius: 5;
}

QComboBox:hover,QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}
</string>
         </property>
         <property name="text">
          <string>Create Room</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="jRoomBtn">
         <property name="minimumSize">
          <size>
           <width>56</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Imprint MT Shadow</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    color: #b1b1b1;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 3px;
    border-color: #3A8524 ;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 38px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

QComboBox
{
    selection-background-color: #ffaa00;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}

QComboBox:hover,QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}
</string>
         </property>
         <property name="text">
          <string>Join Room</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QPushButton" name="goBackBtn_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Image/chees_images/icons8-back-arrow-48.png) ;
background-repeat: no-repeat;
background-position: center;</string>
       </property>
       <property name="text">
        <string> </string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>581</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="getID">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Room ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Image/chees_images/submit.png) ;
background-repeat: no-repeat;
background-position: center;</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QPushButton" name="goBackBtn_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Image/chees_images/icons8-back-arrow-48.png) ;
background-repeat: no-repeat;
background-position: center;</string>
       </property>
       <property name="text">
        <string> </string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>511</width>
         <height>225</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Ravie</family>
            <pointsize>28</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Room Id</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="setID">
          <property name="font">
           <font>
            <family>Wide Latin</family>
            <pointsize>28</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
